tests/Unit/Evolution/Initialization/Test_SetVariables.cpp

```diff
@@ -106,8 +107,11 @@ struct SystemAnalyticSolution : public MarkAsAnalyticSolution {
     return vars;
   }
 
+  EquationsOfState::PolytropicFluid<true> equation_of_state_{100.0, 2.0};
```

Comment:
I'm a bit confused by the two different implementations. The one below returns `equation_of_state_` in the `equation_of_state()` function

Reply:
Yeah this didn't make sense...